packages/test/snapshots/src/validateSnapshots.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import fs from "fs";
+import { assert } from "@fluidframework/common-utils";
+import { FileStorageDocumentName } from "@fluidframework/file-driver";
+import {
+    IFileSnapshot,
+    StaticStorageDocumentServiceFactory,
+} from "@fluidframework/replay-driver";
+import { compareWithReferenceSnapshot, getNormalizedFileSnapshot, loadContainer } from "@fluid-internal/replay-tool";
+import { SnapshotStorageService } from "./snapshotStorageService";
+
+/**
+ * Validates snapshots in the source directory with corresponding snapshots in the destination directory:
+ * - Loads a new container with each snapshot in `srcDir`.
+ * - Snapshots the continer and validates that the snapshot matches with the corresponding snapshot in `destDir`.
+ * @param srcDir - The directory containing source snapshots that are to be loaded and validated.
+ * @param destDir - The directory containing destination snapshots against which the above snaphost are validated.
+ */
+export async function validateSnapshots(srcDir: string, destDir: string) {
+    const errors: string[] = [];
+    // Error handler that reports errors if any while validation.
+    const reportError = (description: string, error?: any) => {
+        let errorString: string;
+        if (error === undefined) {
+            errorString = description;
+        } else if (error instanceof Error) {
+            errorString = `${description}\n${error.stack}`;
+        } else {
+            errorString = `${description} ${error}`;
+        }
+
+        errors.push(errorString);
+        const errorsToReport = 5;
+        if (errors.length <= errorsToReport) {
+            console.error(errorString);
+        } else if (errors.length === errorsToReport + 1) {
+            console.error("\n!!! Too many errors - stopped reporting errors !!!");
+        }
+    };
+
+    for (const node of fs.readdirSync(srcDir, { withFileTypes: true })) {
```

Comment:
rather than node can you call this file, not is just a bit overloaded in our code base which is confusing here